README.md

```diff
@@ -36,10 +36,39 @@ Interested in learning more about the basics of virtual environments in Python
 [article](http://www.cs.allegheny.edu/sites/gkapfham/programming/research/idea/2017/07/14/Virtual-Environments/)
 to further develop your understanding of this topic.
 
+GatorGrouper requires user to use Python3. You can type `python --version` to check
+the current Python version on your devices. If you do not have the right Python version,
+or you can not download or upgrade Python on your devices, you can download
+[Pyenv](https://github.com/pyenv/pyenv) tool to set up the virtual environment of
+the newest Python version.
+
+To install Pyenv, you can use [Pyenv Installer](https://github.com/pyenv/pyenv-installer).
+By typing the command in terminal:
+
+```shell
+curl https://pyenv.run | bash
+```
+
+You will successfully install Pyenv. Please make sure that you have the
+following lines in your `~/.bashrc` file.
+
+```shell
+export PATH="~/.pyenv/bin:$PATH"
+eval "$(pyenv init -)"
+eval "$(pyenv virtualenv-init -)"
+```
+
+Once Pyenv is installed, you can install Python 3.6.7 for your
```

Comment:
This should perhaps use `a version` instead of a specific version, or something similar -- the latest version of Python might change in the future!

Reply:
Solved by https://github.com/GatorEducator/gatorgrouper/pull/123/commits/ba53f0a814a745115bf3c57d3ca2672817db2bd9

Reply:
You should also reference the correct versions in the bash examples as well. Perhaps by just saying that the newest version number should be used to replace `3.6.7` or something similar.